tests/e2e/utils/helpers.ts

```diff
@@ -0,0 +1,30 @@
+import axios from 'axios'
+import { xml2js } from 'xml-js'
+import { _ } from 'lodash'
+import config from '../config'
+
+const sendRequest = ({ method, path }): Promise<any> => {
+  const headers = {
+    Authorization: `Basic ${Buffer.from(`${config.adminUser}:${config.adminPassword}`).toString('base64')}`
+  }
+  return axios({
+    method,
+    url: path.join(config.baseUrlOcis, path),
+    headers
+  })
+}
+
+export const deleteAllFiles = async (): Promise<void> => {
+  // check path, it doesn't seem like this is deleting all files
+  const response = await sendRequest({ method: 'PROPFIND', path: 'remote.php/dav/files/admin' })
+  const xmlResponse = response.data
+  const result = xml2js(xmlResponse, { compact: true })
+  const resp = _.get(result, 'd:multistatus.d:response')
+  const href = _.get(resp[1], 'd:href._text')
+
+  return await sendRequest({ method: 'DELETE', path: href })
```

Comment:
you have to loop through all available hrefs (except the root entry) to delete all files

Reply:
added the loop but still have an Axios error somewhere in the After step. I think the error is triggered by the first call of sendRequest() in deleteAllFiles() of helpers.ts

```
✖ After # tests/e2e/hooks.ts:33
       AxiosError: Request failed with status code 401
           at settle (/Users/k2/JankariTech/web-app-3dmodel-viewer/node_modules/.pnpm/axios@1.6.7/node_modules/axios/lib/core/settle.js:19:12)
           at IncomingMessage.handleStreamEnd (/Users/k2/JankariTech/web-app-3dmodel-viewer/node_modules/.pnpm/axios@1.6.7/node_modules/axios/lib/adapters/http.js:589:11)
           at IncomingMessage.emit (node:events:525:35)
           at IncomingMessage.emit (node:domain:489:12)
           at endReadableNT (node:internal/streams/readable:1359:12)
           at processTicksAndRejections (node:internal/process/task_queues:82:21)
           at Axios.request (/Users/k2/JankariTech/web-app-3dmodel-viewer/node_modules/.pnpm/axios@1.6.7/node_modules/axios/lib/core/Axios.js:45:41)
           at processTicksAndRejections (node:internal/process/task_queues:95:5)
           at async deleteAllFiles (/Users/k2/JankariTech/web-app-3dmodel-viewer/tests/e2e/utils/helpers.ts:24:20)
           at async World.<anonymous> (/Users/k2/JankariTech/web-app-3dmodel-viewer/tests/e2e/hooks.ts:34:3)
```

Reply:
Artur helped me to fix the issue, will commit the refactored code after cleaning up a few things